src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/2.diff.sql

```diff
@@ -237,3 +237,28 @@ AS
    
     COMMIT TRANSACTION
 GO
+
+--
+--  STORED PROCEDURE
+--      SelectMaxSupportedSchemaVersion
+--
+--  DESCRIPTION
+--      Selects the highest supported completed schema version
+--  PARAMETERS
+--      @version
+--          * The version number
+--
+--  RETURNS
+--      The highest version
+--
+CREATE PROCEDURE dbo.SelectMaxSupportedSchemaVersion
```

Comment:
Should be in V3?

Reply:
Yes, since 3.sql was not merged, so kept it in V2. I will move it to V3.